Resources/public/webcomponents/ez-universal-discovery.html

```diff
@@ -0,0 +1,47 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<link rel="import" href="../../../webcomponents/ez-yui-app.html">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery.css">
```

Comment:
for readability, please do not mix stylesheets and import. Also, stylesheets should be imported after the `<style>` element